app/assets/stylesheets/application.scss

```diff
@@ -4,8 +4,7 @@
 @import "rails_bootstrap_forms";
 
 // Geek theme (https://geeksui.codescandy.com/)
-@import 'theme.min.css';
-@import 'feather.css';
+@import 'theme.min.scss';
```

Comment:
Dartsass compiles this `theme.min.scss` and puts it in app/assets/builds which we allow access via propshaft